packages/firestore/src/core/firestore_client.ts

```diff
@@ -72,6 +70,23 @@ const DOM_EXCEPTION_ABORTED = 20;
 /** The DOMException code for quota exceeded. */
 const DOM_EXCEPTION_QUOTA_EXCEEDED = 22;
 
+export class IndexedDbPersistenceSettings {
+  constructor(
+    readonly cacheSizeBytes: number,
+    readonly experimentalTabSynchronization: boolean
+  ) {}
+
+  lruParams(): LruParams {
+    return LruParams.withCacheSize(this.cacheSizeBytes);
+  }
+}
+
+export class MemoryPersistenceSettings {}
```

Comment:
Just to verify my own understanding, this will potentially have cacheSizeBytes in the future (if we move MemoryPersistence to LRU to put web in parity with iOS / Android behavior)?